src/kOS/UserIO/TelnetMainServer.cs

```diff
@@ -143,18 +170,26 @@ private void SetPermanentRealIPPermission(bool newValue)
         
         public void SetConfigEnable(bool newVal)
         {
+            if (bindAddr == null) // when nothing has been set yet, this will be the case.
```

Comment:
`bindAddr` is only set as part of `StartListening` or by selecting an IP address from the toolbar window.  This means that this line will prevent the server from being started unless you manually select an IP address first.  My test case was to open the game and try and toggle telnet on, but every time I clicked the button it would just clear itself.  After I manually selected the loop back address (which the window claimed was currently selected anyways) the toggle worked properly.  My recommendation is that we add `SetBindAddrFromString(SafeHouse.Config.TelnetIPAddrString)` into the `Start` method.  That presents other issues in that `SafeHouse.Config` may still be null (both `Bootstrapper` and `TelnetMainServer` are `StartInstantly`).  I would either move `TelnetMainServer` to start at the `MainMenu` or move `Bootstrapper`'s logic into `Awake`.  I tested the `MainMenu` solution and it worked fine in my use.